### DISCUSSION TITLE
Solving a problem with indirect measurements

### DISCUSSION THREAD

**Discussion by tito21** [lululxvi/deepxde#955] [Q&A]
Hi, I am trying to solve an inverse problem where the measurements are not done in the hidden function but through an operation. More concretely I have a set of measurements $\{S_i, t_i\}$ that are related to the hidden function $u(x, t)$ by:

$$
S(t) = \int u(x,t) dx
$$

There is of course a PDE that has to be satisfied by $u(x, t)$ and has a hidden parameter $\lambda(x)$ that depends on space that I would also like to estimate. 

In the examples, there is no problem like this and by reading the documentation and code I can't figure a way to implement this. I thought of using a costume loss function, but it needs to have a signature `(y_true, y_pred)` and I need access to the model to compute the integral.

**Comment by lululxvi** [comment]
There are similar (but not exactly the same) examples at https://github.com/lululxvi/hpinn

**Reply by tito21** [reply]
Thanks for your answer, I will look into that examples
